src/main/java/org/gridsuite/filter/expertfilter/expertrule/CombinatorExpertRule.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.filter.expertfilter.expertrule;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Identifiable;
+import lombok.AllArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.filter.FilterLoader;
+import org.gridsuite.filter.identifierlistfilter.FilterEquipments;
+import org.gridsuite.filter.utils.expertfilter.CombinatorType;
+import org.gridsuite.filter.utils.expertfilter.DataType;
+
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
+ */
+@AllArgsConstructor
```

Comment:
It's technically a `@NoArgsConstructor` as there is no field.

Reply:
optional

Reply:
Done